src/pages/home/uploadSong/BasicSongDetails.tsx

```diff
@@ -112,11 +122,32 @@ const BasicSongDetails: FunctionComponent = () => {
         } }
       >
         <Stack ref={ audioRef } spacing={ 0.5 } width="100%">
-          <Typography color="grey100" fontWeight={ 500 }>
-            SONG FILE
-          </Typography>
+          { isInEditMode ? (
+            <>
+              <Typography color="grey100" fontWeight={ 500 }>
+                SONG
+              </Typography>
 
-          <UploadSongField name="audio" />
+              <SolidOutline
+                sx={ {
+                  display: "flex",
+                  justifyContent: "center",
+                  alignItems: "center",
+                  flexGrow: 1,
+                } }
+              >
+                <PlaySong id={ songId || "" } />
```

Comment:
Not a major item right now, unless a quick fix, but the responsive view of this box spans the width of the page above ~430px